tidy_data/tidy_data.md

```diff
@@ -0,0 +1,737 @@
+| 2000 | 3 Doors Down | 3:53 | Kryptonite | 2000-04-08 | 1 | 81 |
+| 2000 | 3 Doors Down | 3:53 | Kryptonite | 2000-04-15 | 2 | 70 |
+| 2000 | 3 Doors Down | 3:53 | Kryptonite | 2000-04-22 | 3 | 68 |
+
+The songs keep repeating because we're really tracking two different topics here:
+
+* Basic information about songs (these characteristics don't change over time)
+* Song popularity / top charts data
+
+If we were to abide by the suggestion that each topic belongs in its own table, we could instead have two tables, one for songs and one for ranking, using the id to connect data from the two tables.
+
+<h4>Song Data</h4>
+
+| id | artist | track | time |
+| -- | ----- | ----- | ----- |
+| 1 | 2 Pac   | Baby Don't Cry | 4:22 |  
+| 2 | 2Ge+her | The Hardest Part of ... | 3:15 |
+| 3 | 3 Doors Down | Kryptonite | 3:53 |
+
+
+<h4>Ranking Data</h4>
+
+| id | date | week | rank |
+| ----  | ---  | ---  | ---- |
+| 1 | 2000-02-26 | 1 | 87 |
+| 1 | 2000-03-04 | 2 | 82 |
+| 1 | 2000-03-11 | 3 | 72 |
+| 2 | 2000-09-02 | 1 | 91 |
+| 2 | 2000-09-09 | 2 | 87 |
+| 2 | 2000-09-16 | 3 | 92 |
+| 3 | 2000-04-08 | 1 | 81 |
+| 3 | 2000-04-15 | 2 | 70 |
+| 3 | 2000-04-22 | 3 | 68 |
+
+This means that we only have to correct a mispelled artist name or incorrect track duration in one place, rather than multiple locations.  This is especially important in SQL databases.  Still, this is more an issue of data storage and ease of updating, rather than cleaning up data for computation.  Consider this advice completely optional!
+
+### One type in multiple tables
+
+Occasionally we also have data that treat the same topic but are stored in separate tables.  For example, if you're participating in a multi-site study, you may have data that is structured identically but arrive in your inbox in separate files. Logically, you could imagine "stacking" this data vertically, and perhaps adding a new column that gives the data collection site:
+
+<h4>Clinic A</h4>
+
+| subject_id | dob | abc_score | xyz_score |
+| ---------- | --- | --------- | --------- |
+| 4123691    | 2009-08-12 | 87 | 4 |
+| 4296120    | 2005-10-04 | 82 | 5 |
+| 2934218    | 2008-03-28 | 70 | 4 |
+| 2098821    | 2008-12-10 | 88 | 3 |
+
+<h4>Clinic B</h4>
+
+| subject_id | dob | abc_score | xyz_score |
+| ---------- | --- | --------- | --------- |
+| 3396811    | 2010-12-29 | 78 | 4 |
+| 0129681    | 2010-01-02 | 80 | 3 |
+| 9766728    | 2007-07-10 | 77 | 3 |
+| 6382922    | 2008-03-20 | 90 | 5 |
+
+<h4>Combined Data</h4>
+
+| subject_id | dob | abc_score | xyz_score | collection_site |
+| ---------- | --- | --------- | --------- | -- |
+| 4123691    | 2009-08-12 | 87 | 4 | A |
+| 4296120    | 2005-10-04 | 82 | 5 | A |
+| 2934218    | 2008-03-28 | 70 | 4 | A |
+| 2098821    | 2008-12-10 | 88 | 3 | A |
+| 3396811    | 2010-12-29 | 78 | 4 | B |
+| 0129681    | 2010-01-02 | 80 | 3 | B |
+| 9766728    | 2007-07-10 | 77 | 3 | B |
+| 6382922    | 2008-03-20 | 90 | 5 | B |
+
+## Quiz: Tidying Data
```

Comment:
I feel like this should be a level 3 header, so it falls within the previous section
```suggestion
### Quiz: Tidying Data
```

Reply:
agree, changed